samples/matter/light_switch/src/shell_commands.cpp

```diff
@@ -0,0 +1,231 @@
+ *    limitations under the License.
+ */
+
+#include "shell_commands.h"
+#include "binding_handler.h"
+#include "light_switch.h"
+
+#include <platform/CHIPDeviceLayer.h>
+
+#ifdef CONFIG_CHIP_LIB_SHELL
+
+using namespace chip;
+using namespace chip::app;
+
+namespace SwitchCommands
+{
+using Shell::Engine;
+using Shell::shell_command_t;
+using Shell::streamer_get;
+using Shell::streamer_printf;
+
+Engine sShellSwitchSubCommands;
+Engine sShellSwitchOnOffSubCommands;
+Engine sShellSwitchGroupsSubCommands;
+Engine sShellSwitchGroupsOnOffSubCommands;
+
+static CHIP_ERROR SwitchHelpHandler(int argc, char **argv)
+{
+	sShellSwitchSubCommands.ForEachCommand(Shell::PrintCommandHelp, nullptr);
+	return CHIP_NO_ERROR;
+}
+
+static CHIP_ERROR SwitchCommandHandler(int argc, char **argv)
+{
+	if (argc == 0) {
+		return SwitchHelpHandler(argc, argv);
+	}
+	return sShellSwitchSubCommands.ExecCommand(argc, argv);
+}
+
+static CHIP_ERROR TableCommandHelper(int argc, char **argv)
+{
+	BindingHandler::PrintBindingTable();
+	return CHIP_NO_ERROR;
+}
+
+namespace Unicast
+{
+
+	static CHIP_ERROR OnOffHelpHandler(int argc, char **argv)
+	{
+		sShellSwitchOnOffSubCommands.ForEachCommand(Shell::PrintCommandHelp, nullptr);
+		return CHIP_NO_ERROR;
+	}
+
+	static CHIP_ERROR OnOffCommandHandler(int argc, char **argv)
+	{
+		if (argc == 0) {
+			return OnOffHelpHandler(argc, argv);
+		}
+		return sShellSwitchOnOffSubCommands.ExecCommand(argc, argv);
+	}
+
+	static CHIP_ERROR OnCommandHandler(int argc, char **argv)
+	{
+		BindingHandler::BindingData *data = Platform::New<BindingHandler::BindingData>();
+		data->EndpointId = LightSwitch::GetInstance().GetLightSwitchEndpointId();
+		data->CommandId = Clusters::OnOff::Commands::On::Id;
+		data->ClusterId = Clusters::OnOff::Id;
+
+		DeviceLayer::PlatformMgr().ScheduleWork(BindingHandler::SwitchWorkerHandler,
+							reinterpret_cast<intptr_t>(data));
+		return CHIP_NO_ERROR;
+	}
+
+	static CHIP_ERROR OffCommandHandler(int argc, char **argv)
+	{
+		BindingHandler::BindingData *data = Platform::New<BindingHandler::BindingData>();
+		data->EndpointId = LightSwitch::GetInstance().GetLightSwitchEndpointId();
+		data->CommandId = Clusters::OnOff::Commands::Off::Id;
+		data->ClusterId = Clusters::OnOff::Id;
+
+		DeviceLayer::PlatformMgr().ScheduleWork(BindingHandler::SwitchWorkerHandler,
+							reinterpret_cast<intptr_t>(data));
+		return CHIP_NO_ERROR;
+	}
+
+	static CHIP_ERROR ToggleCommandHandler(int argc, char **argv)
+	{
+		BindingHandler::BindingData *data = Platform::New<BindingHandler::BindingData>();
+		data->EndpointId = LightSwitch::GetInstance().GetLightSwitchEndpointId();
+		data->CommandId = Clusters::OnOff::Commands::Toggle::Id;
+		data->ClusterId = Clusters::OnOff::Id;
+
+		DeviceLayer::PlatformMgr().ScheduleWork(BindingHandler::SwitchWorkerHandler,
+							reinterpret_cast<intptr_t>(data));
+		return CHIP_NO_ERROR;
+	}
+} // namespace Unicast
```

Comment:
Also fix the comment style.

Reply:
In short apply it in every place that you use // :smile:

Reply:
I have a simple script that you can run in `samples/matter` directory to do some fixes to apply the NCS style:
```
#!/bin/bash

function all_files() {
  git ls-files . | grep -v '/zap-generated/'
}

function source_files() {
	all_files | grep -P '\.(h|cpp)$'
}

# fix file names
source_files | while read f; do
  dir="$(dirname "$f")"
  old_name="$(basename "$f")"
  new_name="$(echo "$old_name" | sed -E 's|(\w)([A-Z][a-z])|\1_\2|g' | tr '[A-Z]' '[a-z]')"
  [[ "$old_name" == "$new_name" ]] && continue
  git mv "$f" "$dir/$new_name"
  sed -i "s|$old_name|$new_name|g" $(all_files)
done

# fix comments

sed -i -E 's|//(.*)$|/*\1 */|g' $(source_files) # // -> /* */
clang-format --style=file -i $(source_files)
```

It changes file names from CamelCase to camel_case, comments to the /* */, and reformats sources.